jira-dependency-graph.py

```diff
@@ -65,14 +70,24 @@ def build_graph_data(start_issue_key, jira, excludes, show_directions, direction
     def get_key(issue):
         return issue['key']
 
-    def create_node_text(issue_key, summary):
+    def get_status_color(status_field):
+        if status_field['statusCategory']['name'].upper() in ['IN PROGRESS']:
+            return 'yellow'
+        elif status_field['statusCategory']['name'].upper() in ['DONE']:
+            return 'green'
+        return 'white'
+
+    def create_node_text(issue_key, summary, status, islink):
         # truncate long labels with "...", but only if the three dots are
         # replacing more than two characters -- otherwise the truncated
         # label would be taking more space than the original.
         if len(summary) > MAX_SUMMARY_LENGTH+2:
             summary = summary[:MAX_SUMMARY_LENGTH] + '...'
         summary = summary.replace('"', '\\"')
-        return '"{}({})"'.format(issue_key, summary.encode('utf-8'))
+        log('node ' + issue_key + ' status = ' + str(status))
+        if islink:
+            return '"{}\\n({})"'.format(issue_key, summary.encode('utf-8'))
```

Comment:
@ntavares why don't we add `href` and the other params for `islink` too? Can you give an example?

Reply:
Hi @pawelrychlik, links (as in arrows) don't really map to an entity in JIRA right? As for the rest, they are "fill" properties, which also don't apply to nodes (I actually didn't verify if the attributes are supported, but you're already managing color elsewhere in the code). Does that make sense? Do you have an use case where have an hyperlink on a link (arrow) would be usefull?

Reply:
Ok, it makes sense, thanks.